package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sonar-scanner-vsts",
-  "version": "0.0.1-SNAPSHOT",
+  "version": "0.0.2-SNAPSHOT",
```

Comment:
Not sure you want this update ?

Reply:
I think it's picking the one that had been updated by Wouter while moving from cix to travis, so that's ok. Thanks for pointing that out !